components/public-api-server/pkg/oidc/service.go

```diff
@@ -273,6 +274,10 @@ func (s *Service) Authenticate(ctx context.Context, params AuthenticateParams) (
 	if idToken.Nonce != params.NonceCookieValue {
 		return nil, fmt.Errorf("nonce mismatch")
 	}
+	err = s.validateRequiredClaims(idToken)
+	if err != nil {
+		return nil, fmt.Errorf("failed to validate required claims: %w", err)
```

Comment:
Do we want to propagate the error here? I ask because we're using a 3rd party library which may now (or in the future) choose to put sensitive information into the error and we'd echo that back to the http response.

Reply:
That information is related to a sign-in flow of a user, initiated by a themselves.

Are asking about sensitive data of users?

Reply:
Yes, but the server may hold additional sensitive data at that point, like the token decryption secrets. Even if the data (when sesitive) is for the user who intiated it, we still shouldn't echo it back